inspire/dojson/hep/schemas/hep-0.0.1.json

```diff
@@ -593,6 +612,45 @@
                 }
             }
         },
+        "succeeding_entry": {
```

Comment:
This is hard to answer if one doesn't have the nr of the marc field this is supposed to replace. I assume this refers to

78502   r       superseding record      reportnumber    display: superseded by


w               recid
        z               ISBN

This refers to a record for which we have a revised version as a separate record, e.g. a new edition of a book. The link to the new record can be done via recid, report number or ISBN

- Annette


On Oct 7, 2015, at 11:15 AM, Jan Åge Lavik <notifications@github.com<mailto:notifications@github.com>> wrote:


In inspire/dojson/hep/schemas/hep-0.0.1.json<https://github.com/inspirehep/inspire-next/pull/400#discussion_r41367868>:

> @@ -593,6 +612,45 @@
>                  }
>              }
>          },
> +        "succeeding_entry": {


Yeah, is this perhaps the reference to merged record? @kaplun<https://github.com/kaplun>

—
Reply to this email directly or view it on GitHub<https://github.com/inspirehep/inspire-next/pull/400/files#r41367868>.